.travis.yml

```diff
@@ -1,5 +1,21 @@
 sudo: false
 language: ruby
 rvm:
-  - 2.3.0
-before_install: gem install bundler -v 1.13.6
+- 2.4
+before_script:
```

Comment:
This stuff is based on https://docs.codeclimate.com/docs/travis-ci-test-coverage